modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/import/OnboardingImportFlow.kt

```diff
@@ -0,0 +1,95 @@
+package au.com.shiftyjelly.pocketcasts.account.onboarding.import
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.compose.composable
+import androidx.navigation.navigation
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP40
+
+object OnboardingImportFlow {
+
+    const val route = "onboardingImportFlow"
+
+    private const val start = "start"
+    private const val castbox = "castbox"
+    private const val otherApps = "otherApps"
+
+    fun NavGraphBuilder.importFlowGraph(navController: NavController) {
+        navigation(
+            route = this@OnboardingImportFlow.route,
+            startDestination = start,
+        ) {
+            composable(start) {
+                OnboardingImportStartPage(
+                    onCastboxClicked = { navController.navigate(castbox) },
+                    onOtherAppsClicked = { navController.navigate(otherApps) },
+                    onBackPressed = { navController.popBackStack() },
+                )
+            }
+
+            composable(castbox) {
+                OnboardingImportCastbox(
+                    onBackPressed = { navController.popBackStack() },
+                )
+            }
+
+            composable(otherApps) {
+                OnboardingImportOtherApps(
+                    onBackPressed = { navController.popBackStack() },
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun NumberedList(vararg texts: String) {
+    ConstraintLayout(Modifier.fillMaxWidth()) {
```

Comment:
My concern with that was that if we ever get something to have 10 items, the `10.` would be a different width from `9.`, and it would look bad because the indents wouldn't line up. 

We're not up to 10 yet on either of these screens, but I was also concerned that some variable-width fonts might display the same issue (for example between `1.` and `2.`) although I wasn't able to actually recreate that problem with fewer than 10 items on my device. It's not obvious which, if any, of the fonts on my device are variable width, but I did find at least one report of a variable-width font issue [elsewhere](https://www.trainerroad.com/forum/t/android-app-variable-width-vs-fixed-width-fonts-for-numbers/57075), so it seems like it's at least possible. 

Those are the concerns that led me to go with the admittedly more complex and harder-to-read (but I hope more robust and flexible) ConstraintLayout solution. Let me know if you're not convinced though. 🙂 

> ...and I can also scroll the content in landscape mode on phone.

This was a miss on my part. I've fixed it in [ea2be92](https://github.com/Automattic/pocket-casts-android/pull/634/commits/ea2be928b379ac17813e43100e0502993c24e0a5) (also fixed it for the welcome page, which also had the issue).

Reply:
> My concern with that was that if we ever get something to have 10 items, the 10. would be a different width from 9., and it would look bad because the indents wouldn't line up.

Ah, I see. We could provide a min-width to the cell displaying the index so that at least the text next to it is not indented differently. But I think you were concerned about the indenting of indexes, which is very meticulous of you! 🥇 I'm good with the existing solution. Feel free to merge the PR.